exec/java-exec/src/test/java/org/apache/drill/test/OperatorFixture.java

```diff
@@ -127,20 +156,26 @@ public OperatorFixture build() {
    * uses the same code generation mechanism as the full Drill, but
    * provide test-specific versions of various other services.
    */
-
-  public static class TestFragmentContext extends BaseFragmentContext {
-
+  public static class MockFragmentContext extends BaseFragmentContext {
```

Comment:
More collisions? Can't image you made this many changes here...

Reply:
I like prefixing Mock classes with **Mock** instead of **Test**.